README.md

```diff
@@ -60,6 +114,9 @@ gittuf trust add-policy-key -k ../keys/root --policy-key ../keys/targets.pub
 gittuf policy init -k ../keys/targets
 
 # Add branch protection rule
+# signing_mechanism should be one of: [gpg]
+# TODO: Are there other signing mechanisms? sigstore?
+# If only the only is supported maybe hardcode for now?
```

Comment:
See above, I think the docs for the manual demo run can be updated to clarify that all it needs is two distinct signing identities, maybe two GPG keys that exist already or not, two sigstore identities, one GPG key and one sigstore identity etc.

Reply:
Also, we specifically chose not to store gpg private keys in the repo for the demo because we didn't want to deal with a good samaritan performing automated scans for exposed secrets / keys.

Reply:
> Also, we specifically chose not to store gpg private keys in the repo for the demo because we didn't want to deal with a good samaritan performing automated scans for exposed secrets / keys.

The `gpg-dir` is external to the `repo` folder, so this should not be an issue. Or maybe I am missing something?

Reply:
I meant in the gittuf/demo repo, not the test `repo` folder created during the demo.